operations/observability/Observability.md

```diff
@@ -0,0 +1,300 @@
+# Install and Upgrade Observability Framework
+
+## Overview
+
+Install and Upgrade Observability Framework creates unified consistent requirements for each product including assertions for Goss health checks, as well metrics and dashboards for health checks.
+The framework also includes a unified consistent method to automatically track Time to Install (TTI) and Time to Upgrade (TTU), as well as error and pattern count across clusters and product streams.
+
+1. Establish a consistent framework for health checks/validation, metrics, and reporting for all product streams for install and upgrade observability.
```

Comment:
I have changed it to unordered list.

Reply:
operations/observability/Observability.md:[14](https://github.com/Cray-HPE/docs-csm/actions/runs/4291913692/jobs/7477701189#step:5:15):1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 3]
operations/observability/Observability.md:15:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 3]
operations/observability/Observability.md:16:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 3]
operations/observability/Observability.md:19:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 3]
operations/observability/Observability.md:20:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 3]
operations/observability/Observability.md:22:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 3]

Having these errors in linting because of these unordered list now.